main.py

```diff
@@ -0,0 +1,322 @@
+        """
+    ALPHABET = ('A', 'T', 'G', 'C', 'N')
+    COMPLEMENT_DICT = {'A': 'T', 'T': 'A', 'G': 'C', 'C': 'G', 'N': 'N',
+                       'a': 't', 't': 'a', 'g': 'c', 'c': 'g', 'n': 'n'}
+
+    def __init__(self, sequence):
+        self.sequence = sequence
+
+    def transcribe(self) -> RNASequence:
+        """Transcribes DNA sequence into RNA sequence. Returns RNASequence object"""
+        return RNASequence(self.sequence.replace('T', 'U').replace('t', 'u'))
+
+
+class AminoAcidSequence(BiologicalSequence):
+    """
+    Class representing amino acid sequences
+
+    Attributes:
+        - ONE_LETTER_ALPHABET: contains valid characters
+            for amino acid sequences in the one-letter entry
+        - THREE_LETTER_ALPHABET: contains valid characters
+            for amino acid sequences in the three-letter entry
+
+    Methods:
+        - check_alphabet: checks if the provided amino acid sequence
+            contains only acceptable letters in one-letter or three-letter code
+        - get_molecular_weight: calculate molecular weight for one-letter amino acid sequence
+            """
+
+    ONE_LETTER_ALPHABET = ('A', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'K', 'L',
+                           'M', 'N', 'P', 'Q', 'R', 'S', 'T', 'V', 'W', 'Y')
+    THREE_LETTER_ALPHABET = ('Ala', 'Cys', 'Asp', 'Glu', 'Phe', 'Gly', 'His',
+                             'Ile', 'Lys', 'Leu', 'Met', 'Pro', 'Gln', 'Arg',
+                             'Ser', 'Thr', 'Val', 'Trp', 'Tyr')
+
+    def __init__(self, sequence: str):
+        self.sequence = sequence
+
+    def check_alphabet(self) -> bool:
+        """Checks if the provided amino acid sequence contains only
+        acceptable letters in one-letter or three-letter code"""
+        sequence_chars = set(self.sequence)
+        return (sequence_chars.issubset(self.ONE_LETTER_ALPHABET)
+                or sequence_chars.issubset(self.THREE_LETTER_ALPHABET))
+
+    def get_molecular_weight(self) -> float:
+        """Calculate molecular weight for one-letter amino acid sequence"""
+        WEIGHT_DICT = {
+            'G': 57.051, 'A': 71.078, 'S': 87.077, 'P': 97.115, 'V': 99.131,
+            'T': 101.104, 'C': 103.143, 'I': 113.158, 'L': 113.158, 'N': 114.103,
+            'D': 115.087, 'Q': 128.129, 'K': 128.172, 'E': 129.114, 'M': 131.196,
+            'H': 137.139, 'F': 147.174, 'R': 156.186, 'Y': 163.173, 'W': 186.210
+        }
+        terminal_h_oh_weight = 18.02
+        weight = (terminal_h_oh_weight +
+                  sum(WEIGHT_DICT[aa] for aa in self.sequence if aa in WEIGHT_DICT))
+        return weight
+
+    def __repr__(self):
+        """Returns representation of the object"""
+        return f"{self.__class__.__name__}: {self.sequence}"
+
+    def __str__(self):
+        return self.sequence
+
+
+# -------------------Telegram logger-----------------------
+load_dotenv('./tg_api.env')
+token = os.getenv('TG_API_TOKEN')
```

Comment:
```suggestion
load_dotenv('./tg_api.env')
TOKEN = os.getenv('TG_API_TOKEN')
```
И я бы наверное вынес это в начало скрипта